.github/workflows/tools_tests.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
     - name: Build Docker image
       run: eval "$BUILD"
     - name: Unit test
-      timeout-minutes: 2
+      timeout-minutes: 3
       run: |
         ${{ env.RUN }} "scons -j$(nproc) --directory=/tmp/openpilot cereal && \
                         apt-get update && \
```

Comment:
can we just remove all this apt stuff instead? only missing package is `libdw-dev` - perhaps it should be in there